app.js

```diff
@@ -22,4 +24,56 @@ app.get('/', (req, res) => {
   res.render('index');
 });
 
+app.get('/beers', (req, res) => {
+  punkAPI
+    .getBeers()
+    .then(responseFromAPI => {
+      const data = {
+        beers: responseFromAPI
+      };
+      console.log(data);
+      res.render('beers', data);
+    })
+    .catch(error => {
+      console.log('error', error);
+    });
+});
+
+app.get('/random-beer', (req, res) => {
+  punkAPI
+    .getRandom()
+    .then(responseFromAPI => {
+      const data = {
+        beer: responseFromAPI
```

Comment:
I would pass just the element, so you could skip the `#each` in the render view

```suggestion
        beer: responseFromAPI[0]
```